webxrlayers-1.bs

```diff
@@ -13,7 +13,7 @@ Mailing List Archives: https://lists.w3.org/Archives/Public/public-immersive-web
 !Participate: <a href="https://lists.w3.org/Archives/Public/public-immersive-web-wg/">Mailing list archive</a>
 !Participate: <a href="irc://irc.w3.org:6665/">W3C's #immersive-web IRC</a>
 
-Editor: Rik Cabanier 106988, Facebook https://oculus.com, cabanier@fb.com
+Editor: Rik Cabanier 106988, Meta https://oculus.com, cabanier@fb.com
```

Comment:
Oculus.com also changed to Facebook.com

Reply:
oh, thank you for pointing! I'll update PRs shortly.